backend/src/chef/chef.service.ts

```diff
@@ -3,26 +3,19 @@ import { Chef } from "../chef/chef.entity";
 import { AppDataSource } from "../data-source";
 import { ChefDetails } from "@shared/types/chef.type";
 import { QueryFailedError } from "typeorm";
-import { DuplicateError } from "src/utils/errors/duplicateError.";
+import { DuplicateError } from "src/utils/errors/duplicate.error";
 
 const chefRepository = AppDataSource.getRepository(Chef);
 
 const saveChef = async (details: ChefDetails) => {
   const { uuid, ...rest } = details;
 
-  try {
-    return await AppDataSource.transaction(async (transaction) =>
-      transaction.save(Chef, {
-        ...rest,
-        ...(uuid !== undefined && { uuid }),
-      })
-    );
-  } catch (error) {
-    if (error instanceof QueryFailedError && (error as any).code === "23505")
-      throw new DuplicateError();
-
-    throw error;
-  }
+  return AppDataSource.transaction(async (transaction) =>
+    transaction.save(Chef, {
+      ...rest,
+      ...(uuid !== undefined && { uuid }),
```

Comment:
```suggestion
      ...(uuid && { uuid }),
```
יש מצב שאפילו זה יעבוד:
      ...({ uuid }),
אם הוא undefined הוא פשוט לא ישים כלום